lib/licenses.nix

```diff
@@ -1019,6 +1019,11 @@ in mkLicense lset) ({
     url = "https://qwt.sourceforge.io/qwtlicense.html";
   };
 
+  rsal2 = {
+    fullName = "Redis Source Available License 2.0";
+    url = "https://redis.com/legal/rsalv2-agreement";
```

Comment:
```suggestion
    url = "https://redis.com/legal/rsalv2-agreement";
    free = false;
```

Reply:
How are you arriving at this suggestion? It was not obvious to me that this is "unfree." Anyone can use this software without paying. They merely cannot commercialize it directly.

Reply:
"Free" here does not mean "gratis".  It's used in the sense of [Free Software](https://en.wikipedia.org/wiki/Free_software), which has a quite specific meaning to regarding what freedoms need to be provided to users for the term to apply, and one of those is the freedom to use the program for any purpose, including direct commercialization.

If you look through the rest of the licenses marked as not free in the list, you'll see that very few of them are about software that has a licensing fee attached, but that they all have conditions that restrict users' freedom to the point that they don't count as free software.

Reply:
Thank you! I've updated it.